starlite/utils/model.py

```diff
@@ -20,6 +21,7 @@ def create_parsed_model_field(value: Type[Any]) -> "ModelField":
 
 
 _dataclass_model_map: Dict[Any, Type[BaseModel]] = {}
+_typeddict_model_map: Dict[Any, Type[BaseModel]] = {}
```

Comment:
We don't really need type specific maps here, do we?

Reply:
Not really